charts/kube-prometheus-stack/values.yaml

```
@@ -3873,6 +3873,13 @@ prometheus:
     ## Otherwise, use prometheus.prometheusSpec.additionalConfig (passed through tpl)
     additionalConfigString: ""
 
+    ## Defines the maximum time that the `prometheus` container's startup probe
+    ## will wait before being considered failed. The startup probe will return
+    ## success after the WAL replay is complete. If set, the value should be
+    ## greater than 60 (seconds). Otherwise it will be equal to 600 seconds (15
+    ## minutes).
+    # maximumStartupDurationSeconds: 120
```

```suggestion
    maximumStartupDurationSeconds: 0
```

if zero, the value is omit, too.